helper.php

```diff
@@ -412,6 +413,7 @@ public function printActionTranslatePage() {
                                              '','',array('readonly'=>'readonly')));
                                              
         $form->addElement(form_makeTextField('title',$target_title,$this->getLang('translated_title'),'',$class));
+        $form->addElement(form_makeTextField('target_id',$target_id,$this->getLang('translated_id'),'',$class));
         $form->addElement(form_makeButton('submit','', $this->getLang('create_translation')));
```

Comment:
I added the checkbox to the form. The default behavior is generating pagename from title.